_assets/ci/Jenkinsfile

```diff
@@ -45,11 +44,20 @@ node('linux') {
             'statusgo-ios-simulator': {
                 sh '''
                     make statusgo-ios-simulator
-                    cd build/bin/statusgo-ios-9.3-framework/
+                    cd build/lib/statusgo-ios-9.3-framework/
                     zip -r status-go-ios.zip Statusgo.framework
                 '''
             }
         )
+
+        parallel (
```

Comment:
We don't want to build 4 versions at the same time? Currently, I understand that first android and iOS versions will be built in parallel and when finished, next linux and macOS will be built.

Reply:
That was my initial attempt, but builds were failing erratically until I reduced the number of builds happening at the same time. So I kept the same two that existed before, and then added the two new desktop builds.